pypsa-to-iamc

```diff
@@ -0,0 +1,634 @@
+            
+            #Fuel :Biomass (CCS)
+                                                                 
+            var[sh]['Fuel consumption|Electricity|Solid bio and waste|Primary solid biomass'] = 0.001*ct*(n.links_t.p0.filter(like ='solid biomass CHP').filter(like =country).sum().sum()
+              *n.links_t.p1.filter(like ='solid biomass CHP').filter(like =country).sum().sum()            
+               /(n.links_t.p1.filter(like ='solid biomass CHP').filter(like =country).sum().sum()+n.links_t.p2.filter(like ='solid biomass CHP').filter(like =country).sum().sum()))
+            var[sh]['Fuel consumption|Electricity|Solid bio and waste'] = var[sh]['Fuel consumption|Electricity|Solid bio and waste|Primary solid biomass']
+            var[sh]['Fuel consumption|Electricty|Biomass'] = var[sh]['Fuel consumption|Electricity|Solid bio and waste']
+            
+            var[sh]['Fuel consumption|Electricity|Solid bio and waste|Primary solid biomass|CCS'] = 0.001*ct*(n.links_t.p0.filter(like ='solid biomass CHP CC').filter(like =country).sum().sum()
+              *n.links_t.p1.filter(like ='solid biomass CHP CC').filter(like =country).sum().sum()            
+               /(n.links_t.p1.filter(like ='solid biomass CHP CC').filter(like =country).sum().sum()+n.links_t.p2.filter(like ='solid biomass CHP CC').filter(like =country).sum().sum()))            
+            var[sh]['Fuel consumption|Electricity|Solid bio and waste|CCS'] = var[sh]['Fuel consumption|Electricity|Solid bio and waste|Primary solid biomass|CCS']
+            var[sh]['Fuel consumption|Electricty|Biomass|CCS'] = var[sh]['Fuel consumption|Electricity|Solid bio and waste|CCS']            
+           
+            #Fuel : Hydrogen
+            
+            var[sh]['Fuel consumption|Electricity|Gases|Hydrogen'] = 0.001*ct*(n.links_t.p0.filter(like ='H2 Fuel Cell').filter(like =country)).sum().sum()
+
+            #Fuel : Oil 
+
+            var[sh]['Fuel consumption|Electricity|Liquids|Fossil'] = 0.001*ct*(n.links_t.p0.filter(like ='oil-').filter(like =country)).sum().sum()
+            
+            #Fuel : Hydrogen 
+
+            var[sh]['Fuel consumption|Hydrogen|Gasses|Fossil|Natural gas'] = 0.001*ct*(n.links_t.p0.filter(like ='SMR').filter(like =country)).sum().sum()
+            var[sh]['Fuel consumption|Hydrogen|Gasses|Fossil|Natural gas|CCS'] = 0.001*ct*(n.links_t.p0.filter(like ='SMR CC').filter(like =country)).sum().sum()
+            var[sh]['Fuel consumption|Hydrogen|Electricity'] = 0.001*ct*(n.links_t.p0.filter(like ='H2 Electrolysis').filter(like =country)).sum().sum()
+            var[sh]['Fuel consumption|Hydrogen'] = var[sh]['Fuel consumption|Hydrogen|Gasses|Fossil|Natural gas']+var[sh]['Fuel consumption|Hydrogen|Electricity']
+          
+            #add fuel consumption for stuff that aren't really fuel (wind,solar,etc.) ?
```

Comment:
No, but they're included in the table (not wind, but solar and hydro) and I thought it could be interpreted as 'Energy in' vs. the 'Energy generation' variable which is 'Energy out'